config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This empty duplicate seems worth deleting.

Reply:
Agreed, this should be removed.